site/content/entry/yuzu-progress-report-jul-2023/index.md

```diff
@@ -0,0 +1,95 @@
+Let's jump right in!
+
+<!--more--> 
+
+# Aliens are real!
+
+Golden to add details of PIKMIN specific fixes and some american UFO propaganda /s.
+
+
+# Per-game configurations - Reimagined!
+
+If you're not a new yuz-er, it is very likely that you've seen and used per-game configurations.
+But incase you haven't yet, per-game configurations are a way to set game specific settings without having to redo your settings for every game you start.
+Imagine if few games require GPU accuracy to be "HIGH" instead of "NORMAL" or if some games work best with "OpenGL" over "Vulkan", with per-game configurations you can easily override these specific settings for just those games.
+
+So, what's new, you ask?
+The brains behind this feature's implementation and our trusty slice of toast, [toastUnlimited](https://github.com/lat9nq/), started working towards a future where yuzu could essentially download a game-specific configuration file, that was curated by the community, and then instantly apply those settings.
+Thus users obtaining the optimal settings for any game without having to tinker with the several configuration options we have today.
+
+Sounds exciting, right? Heck yeah!
+While toast was finalizing his designs, he quickly ran into a road-block of his own design that is our current per-game configuration system.
+The existing system's design made it difficult to programmatically define and override each setting in multiple places across yuzu's backend and frontend systems.
+
+toast decided to take the challenge head-on and {{< gh-hovercard "10839" "rewrote the entire settings backend" >}} to improve it while making it programmatically easier to define and implement any setting.
+Thanks to these changes, developers now only need to write out the UI text for a setting and the system will do the rest (reading, writing, representing in the UI, resetting global state).
+
+Note: These changes are currently for developers only with no end-user impact, but do keep your eyes open for follow-up changes that will build on top of this foundation.
+
+
+# Graphics Changes
+
+It's July and We have another new AMD GPU driver with yet another extension broken.
+If you recall from our June progress report, we reported that latest AMD drivers had broken a Vulkan extension - `extendedDynamicState3ColorBlendEquation`, and we had to temporarily disable usage of this extension on AMD driver version `23.5.2` and above.
+
+Fast-forward to July, and it's still broken, kind of.
+Giving credit where due, AMD ***did*** fix this issue with driver version `23.7.2` but only for `RDNA2`, with at least `Polaris` confirmed to be broken.
+In light of this, We {{< gh-hovercard "11182" "reverted" >}} {{< gh-hovercard "11163" "our revert" >}} to {{< gh-hovercard "10946" "our original change from June" >}} and have {{<gh-hovercard "11204" "currently disabled usage of this extension on all AMD drivers" >}}, until AMD fixes this.
```

Comment:
In light of this, and the fact the fixed driver still informs the same Vulkan version, we {{< gh-hovercard "11182" "reverted" >}} {{< gh-hovercard "11163" "our revert" >}} to {{< gh-hovercard "10946" "our original change from June" >}} and have {{<gh-hovercard "11204" "currently disabled usage of this extension on all AMD drivers" >}}, until AMD fixes this.